src/embeddedExplorer/examples/react-example/package.json

```diff
@@ -9,8 +9,8 @@
   "dependencies": {
   },
   "alias": {
-    "react": "../../../node_modules/react",
-    "react-dom": "../../../node_modules/react-dom",
+    "react": "../../../../node_modules/react",
+    "react-dom": "../../../../node_modules/react-dom",
```

Comment:
how was this working before? 🤯

Reply:
it was broken since I moved things into their own folders! Its just the example app for local testing